src/components/CallView/CallView.vue

```diff
@@ -261,6 +297,10 @@ export default {
 			return this.$store.getters.isViewerOverlay
 		},
 
+		isStripeOpen() {
```

Comment:
Works OK for a couple of participants, but:
- If there are more than you screen can fit, presenter may not be in the visible area
- Still a bug, when you're following participant with a screenshare and don't see his video https://github.com/nextcloud/spreed/issues/8799
![Screenshot from 2023-10-10 10-41-54](https://github.com/nextcloud/spreed/assets/93392545/98d1e507-7b28-4f77-ac5a-5a60555c63aa)

So I'd suggest to not hide a button, when stripe is open